### PR TITLE
Update eslint instructions with recommended

### PR DESCRIPTION
**Pull request by AhsanAyaz**
Updated the instructions for both JavaScript and TypeScript

### CODE REVIEW

File: JavaScript/README.md [hunk 1]
```diff
@@ -1,23 +1,29 @@
 # Linting at Salt - JavaScript
 
-Hi again! At Salt you're expected to set up a linter in each and every project. The reason why we insist on always using linters is that being confident with these tools is often expected at work places.
+Hi again! At Salt you're expected to set up a linter for each project. The reason why we insist on always using linting tools to be confident with these tools as these are often used at in companies.
```
Comment (roelineburger) [inline JavaScript/README.md]:
```suggestion
Hi again! At Salt, you're expected to set up a linter for each project. The reason why we insist on always using linting tools is to be confident with these tools as these are often used in companies.
```

File: JavaScript/README.md [hunk 2]
```diff
@@ -1,23 +1,29 @@
...
 
 Unless instructed otherwise, the minimum linting setup that you're expected to use in your projects is the following:
 
-- [The ESLint tool](https://eslint.org/docs/user-guide/getting-started), combined with
-- the AirBnb configuration
+- [The ESLint tool](https://eslint.org/docs/user-guide/getting-started), using the `eslint:recommended` settings
 
-To create this basic linting setup, for running vanilla JavaScript, run the following command in the root of your project: `npm install eslint eslint-config-airbnb eslint-plugin-import eslint-plugin-react eslint-plugin-react-hooks --save-dev` (you can inspect the package.json in one of the correct directory for you to see that we're using this configuration). Then copy the `.eslintrc.json` file from this repo and its contents. Now all you need to do is to create the lint commands described in the `package.json` from this repo.
+To create this basic linting setup, for running vanilla JavaScript, run the following command in the root of your project: `npm init @eslint/config` (you can inspect the package.json in one of the correct directory for you to see that we're using this configuration).
```
Comment (roelineburger) [inline JavaScript/README.md]:
```suggestion
To create this basic linting setup, for running vanilla JavaScript, run the following command in the root of your project: `npm init @eslint/config` (you can inspect the package.json in one of the correct directories for you to see that we're using this configuration).
```

File: JavaScript/README.md [hunk 3]
```diff
@@ -1,23 +1,29 @@
...
+This repository demonstrates a very basic nodejs based application. And we use ESLint in it using the same instructions as above. Install the dependencies with `npm install` and try out the scripts `npm run lint` and `npm run lint-watch`, respectively. Note that the `npm run lint-watch` command is dependent on nodemon which is a developer tool. The **watch** mode helps us keep the lint server up and running so we don't have to restart it as we're fixing the problems. Compare the results in of the terminal output with the comments in the`index.js` file to see if you managed to run the linter correctly. Also, make sure your VSCode shows the same errors using the [ESLint VSCode Extension](https://marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint)
```
Comment (roelineburger) [inline JavaScript/README.md]:
```suggestion
This repository demonstrates a very basic nodejs based application. And we use ESLint in it using the same instructions as above. Install the dependencies with `npm install` and try out the scripts `npm run lint` and `npm run lint-watch`, respectively. Note that the `npm run lint-watch` command is dependent on nodemon which is a developer tool. The **watch** mode helps us keep the lint server up and running so we don't have to restart it as we're fixing the problems. Compare the results of the terminal output with the comments in the`index.js` file to see if you managed to run the linter correctly. Also, make sure your VSCode shows the same errors using the [ESLint VSCode Extension](https://marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint)
```

File: TypeScript/README.md [hunk 4]
```diff
@@ -4,20 +4,64 @@ Hi again! At Salt you're expected to set up a linter in each and every project.
...
+This repository demonstrates a very basic nodejs based application with TypeScript. And we use ESLint in it using the same instructions as above. Install the dependencies with `npm install` and try out the scripts `npm run lint` and `npm run lint-watch`, respectively. Note that the `npm run lint-watch` command is dependent on nodemon which is a developer tool. The **watch** mode helps us keep the lint server up and running so we don't have to restart it as we're fixing the problems. Compare the results in of the terminal output with the comments in the`index.ts` file to see if you managed to run the linter correctly. Also, make sure your VSCode shows the same errors using the [ESLint VSCode Extension](https://marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint)
```
Comment (roelineburger) [inline TypeScript/README.md]:
```suggestion
This repository demonstrates a very basic nodejs based application with TypeScript. And we use ESLint in it using the same instructions as above. Install the dependencies with `npm install` and try out the scripts `npm run lint` and `npm run lint-watch`, respectively. Note that the `npm run lint-watch` command is dependent on nodemon which is a developer tool. The **watch** mode helps us keep the lint server up and running so we don't have to restart it as we're fixing the problems. Compare the results of the terminal output with the comments in the`index.ts` file to see if you managed to run the linter correctly. Also, make sure your VSCode shows the same errors using the [ESLint VSCode Extension](https://marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint)
```

File: TypeScript/package.json [hunk 5]
```diff
@@ -4,21 +4,20 @@
   "description": "A barebones application with a ESLint setup configured with the AirBnb configuration",
```
Comment (roelineburger) [inline TypeScript/package.json]:
```suggestion
  "description": "A barebones application with an ESLint setup",
```

File: TypeScript/package.json [hunk 6]
```diff
@@ -4,21 +4,20 @@
...
   "author": "",
   "license": "ISC",
   "devDependencies": {
-    "@typescript-eslint/eslint-plugin": "^6.3.0",
-    "@typescript-eslint/parser": "^6.3.0",
-    "eslint": "^8.46.0",
+    "@typescript-eslint/eslint-plugin": "^6.4.1",
+    "@typescript-eslint/parser": "^6.4.1",
+    "eslint": "^8.47.0",
     "eslint-config-airbnb-typescript": "^17.1.0",
```
Comment (roelineburger) [inline TypeScript/package.json]: We should remove all the ESLint configurations related to Airbnb.

**Review by roelineburger — APPROVED** [review]
In the JS folder package.json we also need to update the description and the devDependancies to remove Airbnb. 

Otherwise this looks good and easy for follow :) Good job!

**Comment by AhsanAyaz** [comment]
> In the JS folder package.json we also need to update the description and the devDependancies to remove Airbnb.
> 
> Otherwise this looks good and easy for follow :) Good job!

Thanks! Done 👍🏼

**Pull request merged**